File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/library/items/media/files/access/parameters/IFileListParameterProvider.java
package com.lasthopesoftware.bluewater.client.browsing.library.items.media.files.access.parameters;

import com.lasthopesoftware.bluewater.client.browsing.library.items.Item;
import com.lasthopesoftware.bluewater.client.browsing.library.items.playlists.Playlist;

/**
 * Created by david on 11/26/15.
 */
public interface IFileListParameterProvider {
	String[] getFileListParameters(Item item);

	String[] getFileListParameters(Playlist playlist);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/items/media/files/access/parameters/IFileListParameterProvider.kt
package com.lasthopesoftware.bluewater.client.browsing.items.media.files.access.parameters

import com.lasthopesoftware.bluewater.client.browsing.items.ItemId
import com.lasthopesoftware.bluewater.client.browsing.items.playlists.Playlist

/**
 * Created by david on 11/26/15.
 */
interface IFileListParameterProvider {
    fun getFileListParameters(itemId: ItemId): Array<String>
    fun getFileListParameters(playlist: Playlist): Array<String>
}
